webapp/src/app/shared/pipes/order-by.pipe.ts

```diff
@@ -48,8 +48,18 @@ export class OrderByPipe implements PipeTransform {
         } catch (e) {
           return 0;
         }
-      });
-    } else if (args.childProperty && args.property) {
+      });   //sorting dates by time,day,months,year
+    } else if(args.childProperty && args.property.includes("Time")){
+        return records.sort(function (a, b){
+          try{
```

Comment:
Add this as a function to a util file, this can be make generic and used across the application if required.